kotlin-checks-test-sources/src/main/kotlin/checks/LiftReturnStatementCheckSample.kt

```diff
@@ -0,0 +1,223 @@
+package checks
+
+class LiftReturnStatementCheckSample {
+
+    fun returnIfElseNoncompliant(value: Int): String {
+        if (value >= 0) { // Noncompliant {{Lift "return" statements from both branches before "if" statement.}}
```

Comment:
Maybe `Put` or `Place` will sound better here. wdyt?

Reply:
Changed "lift" to "move". Fixed.